src/game/C4GameScript.cpp

```diff
@@ -109,40 +109,13 @@ static long FnGetGravity(C4PropList * _this)
 	return fixtoi(::Landscape.GetGravity() * 100);
 }
 
-static C4String *FnGetPlayerName(C4PropList * _this, long player_nr)
+static C4PropList *FnGetPlayer(C4PropList * _this, long player_nr)
 {
 	if (!ValidPlr(player_nr))
```

Comment:
I think this check is not really necessary, because `::Players.Get(player_nr)` already returns a `nullptr` if that player does not exist